lib/runtime/wasmer/test_helpers.go

```diff
@@ -45,6 +46,17 @@ func NewTestInstanceWithTrie(t *testing.T, targetRuntime string, tt *trie.Trie)
 	return r
 }
 
+func mustHexTo64BArray(t *testing.T, inputHex string) (outputArray [64]byte) {
+	t.Helper()
+	copy(outputArray[:], common.MustHexToBytes(inputHex))
+	return outputArray
+}
+
+func ed25519PubKeyTo32BArray(pubKey ed25519.PublicKey) (array [32]byte) {
```

Comment:
`PublicKeyLength` is 32 bytes anyway, so you dont need this function. just do `pubKey.AsBytes()`

Reply:
Nice catch!